src/os/bluestore/BlueStore.cc

```diff
@@ -19141,19 +19144,12 @@ int BlueStore::read_allocation_from_drive_for_bluestore_tool(bool test_store_and
       } else {
 	derr << "******Failed restore_allocator******\n" << dendl;
       }
-      delete alloc2;
     } else {
       derr << "Failed allcoator2 create" << dendl;
     }
   }
```

Comment:
https://github.com/ceph/ceph/pull/44563/files#diff-6f3b1a5dcf5313dc06f71ba4997ee28ec219ecb5633a728b916465c684f1d87dR19125-R19150
This section is a mixed style of if...else and early return patterns. Can it be made consistent?